src/shared/mount-util.h

```diff
@@ -11,6 +11,15 @@
 #include "errno-util.h"
 #include "macro.h"
 
+typedef enum MountAttrPropagationType {
```

Comment:
maybe add comments here what this maps to, i.e. MS_SHARED, MS_SLAVE, MS_PRIVATE and so on. otherwise people might wonder how this differs from the MS_xyz flags. i.e. maybe just:

```c
typedef enum MountAttrPropagationType {
        PROPAGATION_INHERIT,        /* no special MS_xyz propagation flag */
        PROPAGATION_PRIVATE,        /* → MS_PRIVATE */
        PROPAGATION_DEPENDENT,      /* → MS_SLAVE */
        PROPAGATION_SHARED,         /* → MS_SHARED */ 
        _PROPAGATION_MAX,
        _PROPAGATION_INVALID = -EINVAL,
} MountAttrPropagationType;